packages/lit-dev-content/site/tutorials/tutorials.11tydata.js

```diff
@@ -15,14 +15,14 @@ const path = require('path');
  * @returns {Promise<Object>} Tutorial data to be consumed by
  *     /tutorials/index.html
  */
-const loadTutorialData = async (dirname) => {
+const loadTutorialData = async (dirname, options = {featured: {position: undefined}}) => {
```

Comment:
Is it worth breaking out the `options` type into a separate interface, and documenting that `featured` is an optional `number` between 0 and 2, inclusive?